src/components/idleMembers/card/index.tsx

```diff
@@ -1,13 +1,13 @@
-import { FC } from 'react';
+import { FC, useState } from 'react';
 import classNames from '@/components/idleMembers/card/card.module.scss';
 
 type Props = {
   idleMemberUserName: string
 }
 
 const Card: FC<Props> = ({ idleMemberUserName }) => {
-  const imageGenerator = (name: string) => `${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}/${name}/img.png`;
-
+  const [assigneeProfilePic, setAssigneeProfilePic] = useState(`${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}${idleMemberUserName}/img.png`);
```

Comment:
Instead of **https://raw.githubusercontent.com/Real-Dev-Squad/website-static/main/members/ankita/img.png**, your code shows **https://raw.githubusercontent.com/Real-Dev-Squad/website-static/main/membersankita/img.png**
You probably missed a **'/'** here.
It should be changed to:
const [assigneeProfilePic, setAssigneeProfilePic] = useState(`${process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL}/${idleMemberUserName}/img.png`);

Reply:
Ohh yeah 😅. Got it